Makefile

```diff
@@ -24,16 +24,13 @@ e2e-tests: build
 	cd package && behave
 
 pylint:
-	echo "Start linting"
-	cd package && isort . &
-	black package/kedro_viz package/tests package/features &
-	pylint --rcfile=package/.pylintrc -j 0 package/kedro_viz &
-	pylint --rcfile=package/.pylintrc -j 0 --disable=protected-access,missing-docstring,redefined-outer-name,no-self-use,invalid-name,too-few-public-methods,no-member,unused-argument,duplicate-code package/tests &
-	pylint --rcfile=package/.pylintrc -j 0 --disable=missing-docstring,no-name-in-module,unused-argument package/features &
-	flake8 --config=package/.flake8 package &
-	mypy --config-file=package/mypy.ini package &
-	wait
-	echo "Done linting"
+	cd package && isort .
+	black package/kedro_viz package/tests package/features
+	pylint --rcfile=package/.pylintrc -j 0 package/kedro_viz
+	pylint --rcfile=package/.pylintrc -j 0 --disable=protected-access,missing-docstring,redefined-outer-name,no-self-use,invalid-name,too-few-public-methods,no-member,unused-argument,duplicate-code package/tests
+	pylint --rcfile=package/.pylintrc -j 0 --disable=missing-docstring,no-name-in-module,unused-argument package/features
+	flake8 --config=package/.flake8 package
+	mypy --config-file=package/mypy.ini package
```

Comment:
I changed this in another PR to parallelise this step but I didn't know `&` and `wait` doesn't work on makefile across serveral lines, so I reverted it here.